### PR TITLE
#6223 Added test proofs working dynamiclly added drivers.

### PR DESCRIPTION
**Pull request by michalgce**

### CODE REVIEW

File: kernel/sql/src/test/java/com/twosigma/beakerx/sql/SQLMySQLEvaluatorTest.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+package com.twosigma.beakerx.sql;
```
Comment (scottdraves) [inline kernel/sql/src/test/java/com/twosigma/beakerx/sql/SQLMySQLEvaluatorTest.java]: banner

File: kernel/sql/src/test/java/com/twosigma/beakerx/sql/SQLMySQLEvaluatorTest.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+public class SQLMySQLEvaluatorTest {
+
+  private static final String SRC_TEST_RESOURCES = "./src/test/resources/";
+  private static final String CLASSPATH_TO_MYSQL_DRIVER_JAR = SRC_TEST_RESOURCES + "jdbc_drivers/mysql-connector-java-6.0.6.jar";
+  private static final String TEST_DB_USER = "minty";
+  private static final String TEST_DB_PASSWORD = "anotherPassword";
+  private static final Integer TEST_DB_PORT = 2215;
```
Comment (scottdraves) [inline kernel/sql/src/test/java/com/twosigma/beakerx/sql/SQLMySQLEvaluatorTest.java]: network config could block this?

File: kernel/sql/src/test/resources/db/mysql.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS `docs` (
```
Comment (scottdraves) [inline kernel/sql/src/test/resources/db/mysql.sql]: banner

File: kernel/sql/build.gradle [hunk 4]
```diff
@@ -52,6 +52,8 @@ dependencies {
   testCompile project(':base').sourceSets.test.output
   testCompile group: 'org.assertj', name: 'assertj-core', version: '3.6.1'
   testCompile group: 'junit', name: 'junit', version: '4.12'
+  testCompile group: 'com.wix', name: 'wix-embedded-mysql', version: '3.0.0'
```
Comment (scottdraves) [inline kernel/sql/build.gradle]: idk if this is a good idea.

**Comment by michalgce** [comment]
@scottdraves
Probably the reason of failure build was missing library in os which runs jenkins. To avoid installing additional libraries I changed implementation of embedded database from mysql to postgresql.

File: kernel/sql/build.gradle [hunk 5]
```diff
@@ -48,10 +51,15 @@ dependencies {
...
   compile group: 'com.h2database', name: 'h2', version: '1.4.193'
+  compile group: 'com.microsoft.sqlserver', name: 'mssql-jdbc', version: '6.2.2.jre8'
+  compile group: 'postgresql', name: 'postgresql', version: '9.1-901-1.jdbc4'
+  compile group: 'mysql', name: 'mysql-connector-java', version: '6.0.6'
+  compile group: 'com.amazon.redshift', name: 'redshift-jdbc42', version: '1.2.10.1009'
 
   testCompile project(':base').sourceSets.test.output
   testCompile group: 'org.assertj', name: 'assertj-core', version: '3.6.1'
   testCompile group: 'junit', name: 'junit', version: '4.12'
+  testCompile group: 'ru.yandex.qatools.embed', name: 'postgresql-embedded', version: '2.5'
```
Comment (scottdraves) [inline kernel/sql/build.gradle]: please remove this is too complex

**Pull request merged**